xbmc/utils/MathUtils.h

```diff
@@ -27,6 +27,7 @@
     defined(__alpha__) || \
     defined(__arc__) || \
     defined(__arm__) || \
+    defined(__loongarch) || \
```

Comment:
Should be `__loongarch__` instead... Mind sending a fix PR?

Reply:
> Should be `__loongarch__` instead... Mind sending a fix PR?
Yes, an typo.

Reply:
https://github.com/xbmc/xbmc/pull/22073